nautobot/extras/models/models.py

```diff
@@ -981,3 +981,15 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+#
+# Health Check
+#
+
+
+class HealthCheckTestModel(models.Model):
```

Comment:
Can/should we inherit from Nautobot's `nautobot.core.models.BaseModel` class instead?